client/src/styles/Header/Header.scss

```diff
@@ -0,0 +1,163 @@
+nav {
+  margin: 0;
+  overflow: hidden;
+  background: black;
+  height: 50px;
+}
+
+.nav-content {
+  max-width: 1024px;
+  margin: 0 auto;
+  padding: 0 15px;
+  display: flex;
+  justify-content: space-between;
+  margin-top: 2px;
+
+  a {
+    display: block;
+    font-size: 14px;
+    line-height: 44px;
+    text-decoration: none;
+    transition: all 0.3s;
+    color: #ffffff;
+  }
+}
+
+.icon {
+  opacity: 0.8;
+  margin-top: 8px;
+
+  &:hover {
+    opacity: 1;
+  }
+}
+
+.nav-links {
+  width: 75%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  a {
+    opacity: 0.8;
+
+    &:hover {
+      opacity: 1;
+    }
+  }
+}
+
+.hra {
+  display: none;
+}
+
+.nav-icon {
+  display: none;
+  grid-gap: 5px;
+  grid-template-columns: auto;
+  margin-top: 2px;
+  padding: 17px 0;
+  height: 10px;
+  cursor: pointer;
+  transition: all 0.3s;
+}
+
+.bar {
+  height: 1px;
+  width: 18px;
+  background: white;
+  transition: 0.5s;
+}
+
+.search {
+  cursor: pointer;
+  transition: all 0.3s;
+  opacity: 0.8;
+  margin-top: 11px;
+
+  &:hover {
+    opacity: 1;
+  }
+}
+
+.search1 {
```

Comment:
`search1`? Why?

Reply:
It's for the bag besides the search icon

Reply:
@nehaparmar99 Can we change it to something like `search-bag`?